src/Schema/ForeignKeyConstraint.php

```diff
@@ -196,6 +196,8 @@ public function getUnquotedForeignColumns()
     /**
      * {@inheritdoc}
      *
+     * @deprecated Use {@link getLocalColumns()} instead.
```

Comment:
Should we adjust the calls in those methods accordingly?

* `Doctrine\DBAL\Schema\SchemaException::namedForeignKeyRequired()`
* `Doctrine\DBAL\Schema\Table::_addForeignKeyConstraint()`

Reply:
No, because this method may be overridden in a custom class extending `ForeignKeyConstraint`. Us no longer calling it may result in a BC break.